internal/tmmsgs/en_error_messges.go

```diff
@@ -85,16 +85,20 @@ var (
-	MsgTransactionHandlerRequestInvalid       = ffe("FF21073", "Invalid transaction handler request type '%d'")
-	MsgTransactionHandlerResponseNoSequenceID = ffe("FF21074", "Transaction handler failed to allocate a sequence ID for request '%s'", http.StatusInternalServerError)
-	MsgPersistenceSequenceIDNotAllowed        = ffe("FF21075", "New transaction should not have sequence ID")
-	MsgInvalidJSONGasObject                   = ffe("FF21076", "Failed to parse response from Gas Oracle REST API as a JSON object")
-
-	MsgTHMetricsInvalidName     = ffe("FF21077", "Transaction handler metrics registration name can only contain lowercase letters and underscore. Actual name: %s")
-	MsgTHMetricsHelpTextMissing = ffe("FF21078", "Transaction handler metrics registration help text must be provided")
-	MsgTHMetricsDuplicateName   = ffe("FF21080", "Transaction handler metrics registration invalid name already registered: %s")
+	MsgTransactionHandlerNotRegistered         = ffe("FF21070", "No transaction handler registered with name '%s'")
+	MsgNoGasConfigSetForTransactionHandler     = ffe("FF21071", "A fixed gas price must be set when not using a gas oracle")
+	MsgTransactionHandlerRequestTimeout        = ffe("FF21072", "The transaction handler did not acknowledge the request after %.2fs", 408)
+	MsgTransactionHandlerRequestInvalid        = ffe("FF21073", "Invalid transaction handler request type '%d'")
+	MsgTransactionHandlerResponseNoSequenceID  = ffe("FF21074", "Transaction handler failed to allocate a sequence ID for request '%s'", http.StatusInternalServerError)
+	MsgPersistenceSequenceIDNotAllowed         = ffe("FF21075", "New transaction should not have sequence ID")
+	MsgInvalidJSONGasObject                    = ffe("FF21076", "Failed to parse response from Gas Oracle REST API as a JSON object")
+	MsgTHMetricsInvalidName                    = ffe("FF21077", "Transaction handler metrics registration name can only contain lowercase letters and underscore. Actual name:%s")
+	MsgTHMetricsHelpTextMissing                = ffe("FF21078", "Transaction handler metrics registration help text must be provided")
+	MsgTHMetricsDuplicateName                  = ffe("FF21080", "Transaction handler metrics registration invalid name already registered: %s")
+	MsgInvalidNonRichQuery                     = ffe("FF21081", "Rich query is not supported by persistence. Query params limited to %v")
+	MsgSQLScanFailed                           = ffe("FF21082", "Failed to read value from DB (%T)")
+	MsgShuttingDown                            = ffe("FF21083", "Connector shutdown initiated", 500)
+	MsgTransactionPersistenceError             = ffe("FF21084", "Failed to persist transaction data", 500)
+	MsgOpNotSupportedWithoutRichQuery          = ffe("FF21085", "Not supported: The connector must be configured with a rich query database to support this operation", 405)
```

Comment:
Not sure this is strictly a case for HTTP 405 (that's usually when you POST something that only supports GET, or similar). HTTP 501 might be more correct?